CMakeLists.txt

```diff
@@ -169,8 +173,10 @@ adios_option(AWSSDK     "Enable support for S3 compatible storage using AWS SDK'
 include(${PROJECT_SOURCE_DIR}/cmake/DetectOptions.cmake)
 
 if(ADIOS2_HAVE_CUDA)
-    set(CMAKE_CUDA_STANDARD 11)
-    set(CMAKE_CUDA_STANDARD_REQUIRED TRUE)
+    if(NOT CMAKE_CUDA_STANDARD)
```

Comment:
@anagainaru this thread was for the same issue as the CMAKE_CXX_STANDARD thread. CMAKE_CUDA_STANDARD is not much of a trouble because it is used in a few TUs but still having this would signal that we we support multiple C++ revisions for files with CUDA functions while not being true.

I'll leave it to you whether to maintain the change if you see that this is needed for the feature you are adding.

Reply:
Ups, I though I changed both, will update the code to remove the CUDA standard as well.